src/Discord/Discord.php

```diff
@@ -1358,6 +1358,25 @@ public function __set(string $name, $value)
         $this->client->{$name} = $value;
     }
 
+    /**
+     *
+     * Gets a channel.
+     *
+     * @param string|int $channel_id Id of the channel.
+     *
+     * @return Channel
+     */
+    public function channelGet($channel_id): ?Channel
```

Comment:
Maybe `getChannel` or `findChannel` instead?

Reply:
the name was inspired by offsetGet

Reply:
I think it should be the other way around, the only reason it is called `offsetGet` is because that is the function name in the PHP interface: https://www.php.net/manual/en/class.arrayaccess.php